applanix_bridge/launch/applanix_bridge.launch

```diff
@@ -3,7 +3,8 @@
   <arg name="publish_tf" default="True"/>
   <arg name="odom_frame" default="gps_odom"/>
   <arg name="base_frame" default="base_link"/>
-  <arg name="zero_start" default="True"/>
+  <arg name="zero_start" default="False"/>
+  <arg name="origin" default="None"/>
```

Comment:
Can you add an example of origin (any lat, long, altitude) provided as an arg too (in place of None) ? It would be good for the user to know how to use it in case of custom start point.